index.html

```diff
@@ -457,6 +457,12 @@ <h3>ecdsa-2019</h3>
 section also include the verification of such a data integrity proof.
         </p>
 
+        <p class="advisement">
+When the RDF Dataset Canonicalization Algorithm [[RDF-CANON]] is utilized,
+implementations MUST detect <a data-cite="RDF-CANON#dataset-poisoning">
+dataset poisoning</a> by default and abort processing.
```

Comment:
```suggestion
implementations MUST ensure that, by default, <a data-cite="RDF-CANON#dataset-poisoning">
poison datasets</a> are detected and not processed.
```
The current wording emphasizes the responsibility of the caller of the RDFC-1.0 algorithm. But in rdf-canon, we encourage implementers of that algorithm to abort when they detect a poison dataset. So most users would only have to check that the RDFC-10 implementation they are using has the appropriate default for them, or set the mitigation parameters. Of course, in some situations, they may need to detect poison graph themselves...

Reply:
@pchampin, I took @TallTed's modification to this section instead of yours because of the phrase you added "are detected". I think that language would require Data Integrity implementations to detect a poison graph /before/ sending it to RDF-CANON, which is designed to detect poison graphs. We don't want to give the impression that poison graphs need to be detected before RDF-CANON is used.

@TallTed's language didn't imply that anything other than RDF-CANON processing needs to be performed.